### ISSUE TITLE
Consider providing UMD modules

### ISSUE THREAD

**Issue by IagoLast** [GoogleChrome/imagecapture-polyfill#3]
Hi, since this is a polyfill I think that it makes sense to provide a UMD module under the lib folder to the file can be loaded as a script in a webpage without transpiling.

Currently if someone downloads the project when its included in a webpage like this

```html
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<script src="node_modules/image-capture/lib/imagecapture.js"></script>
</head>

<body>
</body>

</html>
```

the following error is thrown:

```
Uncaught SyntaxError: Unexpected token export
```

While the [package.json](https://github.com/GoogleChrome/imagecapture-polyfill/blob/master/package.json) gives 2 different I think that the lib version should be published like an UMD module, while the src can be kept as an esnext module.

Making this change is as easy as updating the package.json file and force babel to use umd modules.

```
"babel": {
    "presets": [
      [
        "es2015",
        {
          "modules":  "umd"
        }
      ]
    ]
  }

```

Regards

**Comment by cdata** [comment]
It would be nice if you could bundle a ready-to-go ES5 version of the library (possibly wrapped in UMD per the above proposal).

As currently structured, this polyfill adds a tooling burden to any project that may want to use the most up to date source, but has made different tooling choices. This burden can add a lot of undesired overhead to e.g., testing environments, or for projects that have not already bought in to source transformation (e.g., most repos in the PolymerElements org).

**Comment by dandv** [comment]
Should be fixed in cd237e5. Thanks for the interest in [the module](https://www.npmjs.com/package/image-capture)!

**Comment by cdata** [comment]
Ah, I see. So you are distributing the built file in npm, then?
